docs/source/installation.rst

```diff
@@ -136,52 +136,47 @@ Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
 
     .. tabs::
 
-        .. tab:: CPU Support
-
-            If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:
+        .. tab:: PyPI Installation
 
             .. tabs::
 
-                .. tab:: Windows/Linux
-
-                    .. code::
+                .. tab:: Nvidia GPU Support
 
-                       pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html
+                    ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
```

Comment:
It might be worthwhile to instruct people to run `nvidia-smi` to check their CUDA and driver versions:

![Screenshot from 2022-01-10 13-16-28](https://user-images.githubusercontent.com/16181459/148817710-565e0b9a-d4bf-4941-91b6-ca926133c1d3.png)

Reply:
Great point! Added two more sentences to provide brief resources to help people out with that.